README.md

```diff
@@ -25,32 +25,34 @@ Installation steps:
 <h3>Configuration:</h3>
 
 From the main menu choose **Plugins --> QRealTime --> QRealTime Setting**
-<br/> Here you have two tabs one for Aggregate and another for KoboToolbox
-Choose one of the tab and Enter url (required). Other fields are optional.
-<br/>
-You can create free account in <a href="https://www.kobotoolbox.org/">KoboToolbox server </a> and if you want to install your own aggregate server, <a href="http://docs.opendatakit.org/aggregate-guide/"> then visit </a>
-<br/>
-<img src="https://user-images.githubusercontent.com/5653512/67921427-aa90e200-fbcd-11e9-874b-fc1fae692fe0.png" alt="settings">
+<br/> Here you have three tabs one for Aggregate, KoboToolBox, and Central
+Choose one of the tabs and Enter url (required). Other fields are optional.
+<br/><br/>
+You can install your own aggregate server <a href="http://docs.opendatakit.org/aggregate-guide/"> here </a> <br/>
```

Comment:
I recommend replacing this by Central and not linking to Aggregate docs. The plugin support will certainly still be helpful for folks who have existing Aggregate servers but we don't recommend setting up a new one.

A follow-on issue would be to swap the order of the Central and Aggregate tabs in the UI.